src/main/java/ca/unknown/bot/commands/ScheduleReminderCommands.java

```diff
@@ -0,0 +1,73 @@
+package ca.unknown.bot.commands;
+
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.Commands;
+import net.dv8tion.jda.api.interactions.commands.build.SlashCommandData;
+
+public class ScheduleReminderCommands {
+    public static SlashCommandData getScheduleExamCommand(){
+        return Commands.slash("schedule-exam",
+                        "Schedules a new exam reminder. Please format your date as " +
+                                "YYYY MM DD HR MIN SEC.")
+                .addOption(OptionType.STRING, "course",
+                        "The course code of the exam.", true)
+                .addOption(OptionType.STRING, "location",
+                        "The location of your exam.", true)
+                .addOption(OptionType.INTEGER, "year",
+                        "The year of your exam.", true)
+                .addOption(OptionType.INTEGER, "month",
+                        "The numeric month of your exam.", true)
+                .addOption(OptionType.INTEGER, "day",
+                        "The numeric day of your exam.", true)
+                .addOption(OptionType.INTEGER, "hour",
+                        "The military hour of your exam.", true)
+                .addOption(OptionType.INTEGER, "minute",
+                        "The minutes value of your exam time.", true)
+                .addOption(OptionType.INTEGER, "sec",
```

Comment:
there should be no seconds command, maybe run a git pull since i removed it from the main and instead made it so that it automatically puts 00 as the seconds in my controller class